cmd/landscaper-controller/app/app.go

```diff
@@ -77,11 +81,32 @@ func (o *Options) run(ctx context.Context) error {
 		opts.MetricsBindAddress = fmt.Sprintf(":%d", o.Config.Metrics.Port)
 	}
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), opts)
+	hostMgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), opts)
 	if err != nil {
 		return fmt.Errorf("unable to setup manager: %w", err)
 	}
 
+	lsMgr := hostMgr
+	if len(o.landscaperKubeconfigPath) > 0 {
+		data, err := ioutil.ReadFile(o.landscaperKubeconfigPath)
+		if err != nil {
+			return fmt.Errorf("unable to read landscaper kubeconfig from %s: %w", o.landscaperKubeconfigPath, err)
+		}
+		client, err := clientcmd.NewClientConfigFromBytes(data)
+		if err != nil {
+			return fmt.Errorf("unable to build landscaper cluster client from %s: %w", o.landscaperKubeconfigPath, err)
+		}
+		hostConfig, err := client.ClientConfig()
```

Comment:
nit: that we do not confuse the host cluster with the landscaper cluster
```suggestion
		lsConfig, err := client.ClientConfig()
```

Reply:
done